ModelImporter.cpp

```diff
@@ -85,12 +85,23 @@ Status importInput(ImporterContext* importer_ctx,
 
 Status importInputs(ImporterContext* importer_ctx,
```

Comment:
Name of this method is confusing, as it seems to import only static inputs AKA weights. Please rename

Reply:
OK.